README.md

```diff
@@ -18,6 +18,16 @@ Arrow GLib is a C wrapper for [Arrow C++](https://github.com/apache/arrow/tree/m
 
 gobject-introspection gem is a Ruby bindings of GObject Introspection. Red Arrow uses GObject Introspection via gobject-introspection gem.
 
+## Before Install
+### Ubuntu
+
+If you want to use Ruby packages for Ubuntu 16.04 LTS. 
+
+```text
+% sudo apt install ruby ruby-dev
+% gem update
```

Comment:
It seems that we need `sudo` for `gem update`:

```console
% gem update
Updating installed gems
Updating bigdecimal
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /var/lib/gems/2.3.0 directory.
```

Can you confirm it again?

Reply:
One more question. We need to update all gems? Or we need to update only `rake` gem?